src/components/one-time-donation/LoginForm.tsx

```diff
@@ -11,6 +11,11 @@ import { StepsContext } from './helpers/stepperContext'
 import { AlertStore } from 'stores/AlertStore'
 import PasswordField from 'components/common/form/PasswordField'
 
+const onGoogleLogin = () => {
+  const resp = signIn('google')
+  console.log(resp)
```

Comment:
I guess the `console.log` can go?

Reply:
aaah, the copy paste - removed from the original place too... and pls don't ask why we have two login forms... this needs to be fixed separately